view/ViewableData.php

```diff
@@ -541,6 +542,9 @@ public function CSSClasses($stopAtClass = 'ViewableData') {
 			$classes[] = $this->template;
 		}
 
+		// Strip out namespaces
+		$classes = preg_replace('#.*\\\\#', '', $classes);
```

Comment:
Unfortunately this won't work for namespaced templates, since those won't exist in the class manifest, but will have a namespace (i.e. path).